cores/cv32e40p/user_manual/source/instruction_set_extensions.rst

```diff
@@ -317,6 +317,16 @@ together.
 -  Immediate branching instructions are useful to compare a register
    with an immediate value before taking or not a branch, see see :ref:`pulp_immediate_branching`.
 
+Extract, Insert, Clear and Set instructions have the following meaning:
```

Comment:
The file seems to use non-standard characters; see image below when I run make html:

![image](https://user-images.githubusercontent.com/40633348/85132859-eef2f000-b239-11ea-9378-7a3f4ffc3c2c.png)

Reply:
I don't have such errors, can you help me to understand what to change?